authors/apps/authentication/urls.py

```diff
@@ -1,8 +1,8 @@
 from django.urls import path
 from .views import (
     LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView,
-    ActivationLinkView
-)
+    PasswordReset, PasswordDone,
+    ActivationLinkView)
```

Comment:
The closing brace should be in a new line as done in the import statement spanning over lines 23 - 26 of the [view.py](https://github.com/andela/ah-the-unsullied/blob/0650c1aff12578aacade3115ebec1ac3a63aee76/authors/apps/authentication/views.py) file